plugins/outputs/sensu/README.md

```diff
@@ -0,0 +1,85 @@
+# HTTP Output Plugin
+
+This plugin writes metrics events to [Sensu Go](https://sensu.io)  via its 
+HTTP events API.
+
+### Configuration:
+
+```toml
+  ## BACKEND API URL is the Sensu Backend API root URL to send metrics to 
+  ## (protocol, host, and port only). The output plugin will automatically 
+  ## append the corresponding backend or agent API path (e.g. /events or 
+  ## /api/core/v2/namespaces/:entity_namespace/events/:entity_name/:check_name).
+  ## 
+  ## NOTE: if backend_api_url and agent_api_url and api_key are set, the output 
+  ## plugin will use backend_api_url. If backend_api_url and agent_api_url are 
+  ## not provided, the output plugin will default to use an agent_api_url of 
+  ## http://127.0.0.1:3031
+  ## 
+  # backend_api_url = "http://127.0.0.1:8080"
+  # agent_api_url = "http://127.0.0.1:3031"
+
+  ## API KEY is the Sensu Backend API token 
+  ## Generate a new API token via: 
+  ## 
+  ## $ sensuctl cluster-role create telegraf --verb create --resource events,entities
+  ## $ sensuctl cluster-role-binding create telegraf --cluster-role telegraf --group telegraf
+  ## $ sensuctl user create telegraf --group telegraf --password REDACTED 
+  ## $ sensuctl api-key grant telegraf
+  ##
+  ## For more information on Sensu RBAC profiles & API tokens, please visit: 
+  ## - https://docs.sensu.io/sensu-go/latest/reference/rbac/
+  ## - https://docs.sensu.io/sensu-go/latest/reference/apikeys/ 
+  ## 
+  # api_key = "${SENSU_API_KEY}"
+  
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+
+  ## Timeout for HTTP message
+  # timeout = "5s"
+
+  ## HTTP Content-Encoding for write request body, can be set to "gzip" to
+  ## compress body or "identity" to apply no encoding.
+  # content_encoding = "identity"
+
+  ## Sensu Event details 
+  ##
+  ## Below are the event details to be sent to Sensu.  The main portions of the
+  ## event are the check, entity, and metrics specifications. For more information
+  ## on Sensu events and its components, please visit:
+  ## - Events - https://docs.sensu.io/sensu-go/latest/reference/events
+  ## - Checks -  https://docs.sensu.io/sensu-go/latest/reference/checks
+  ## - Entities - https://docs.sensu.io/sensu-go/latest/reference/entities
+  ## - Metrics - https://docs.sensu.io/sensu-go/latest/reference/events#metrics
+  ## 
+  ## Check specification
+  ## The check name is the name to give the Sensu check associated with the event
+  ## created.
+  [outputs.sensu-go.check]
+    name = "telegraf"
```

Comment:
So does this relate to the `metadata/name` property in the documentation you linked?

Reply:
Yes.

Reply:
Ok, so probably everybody using this plugins knows. If you like you might want to add a short notice that the properties are identical to the metadata ones in the documentation!?

Reply:
Mark as resolved.